mqtt3/1.0.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,182 @@
+* Publish messages with retention.
+* Publish messages to an MQTT topic and set different Quality of Service (QoS) levels.
+
+== Configure the Connector
+
+MQTT3 Connector comes out of the box with a set of default values for both publishing and consuming messages. The only requirement is that you must configure which connection (`<mqtt3:connection>`) to use and that you specify a value for the client ID (`clientID`), which will uniquely identify that connection.
+
+In the following example, you configure a minimal connection to a broker. The `<mqtt3:connection>` and `clientID` configurations look like this:
+
+[source,xml,linenums]
+----
+<mqtt3:config name="MQTT_Config">
+    <mqtt3:connection clientId="smart-bentley-123">
+        <mqtt3:connection-settings>
+            <mqtt3:connection-string-mqtt3-authenticator url="tcp://127.0.0.1:1883" />
+        </mqtt3:connection-settings>
+    </mqtt3:connection>
+</mqtt3:config>
+----
+
+If you want to specify each URL field separately, set the `<mqtt3:form-mqtt3-authenticator>` for the connection:
+
+[source,xml,linenums]
+----
+<mqtt3:config name="MQTT3_Config">
+	<mqtt3:connection clientId="smart-bentley-123" >
+		<mqtt3:connection-settings >
+			<mqtt3:form-mqtt3-authenticator host="0.0.0.0" protocol="TCP" port="1883"/>
+		</mqtt3:connection-settings>
+	</mqtt3:connection>
+</mqtt3:config>
+----
+
+=== Define Global Defaults
+
+MQTT3 Connector enables you to define multiple default parameters while consuming or publishing messages. This way, you can define a global default behavior for all the operations associated with the configurations.
+
+In the following example, you configure the keep alive internal time units (`keepAliveInterval` and `keepAliveIntervalUnit`) to set the maximum period of time that the connection is kept alive without any messages being exchanged between the client and broker. You also configure the maximum number of in-flight (`maxInFlight`) messages allowed:
+
+[source,xml,linenums]
+----
+<mqtt3:config name="MQTT_Config">
+    <mqtt3:connection clientId="smart-bentley-123" maxInFlight="60" keepAliveInterval="60" keepAliveIntervalUnit="SECONDS">
+        <mqtt3:connection-settings >
+            <mqtt3:connection-string-mqtt3-authenticator url="tcp://127.0.0.1:1883" />
+        </mqtt3:connection-settings>
+    </mqtt3:connection>
+</mqtt3:config>
+----
+
+
+== Set a Client ID
+
+The client ID is mandatory because it identifies an MQTT connection to a broker. Define a meaningful name that uniquely identifies a client or device that connects to an MQTT broker and not a random string. Set the client ID (`clientId`) in the connection element (`<mqtt3:connection>`)  of the configuration.
+
+== Specify a Connection Protocol
+
+MQTT supports protocols, which can be used to connect to and exchange MQTT messages with the broker. Configure the protocol in the connection string of the connector configuration and will default to TCP if none is specified.
```

Comment:
The last sentence is ungrammatical. Maybe say this? Configure the protocol in the connection string of the connector configuration. The default is TCP.